crates/pathfinder/src/core.rs

```diff
@@ -28,6 +28,10 @@ pub struct ContractHash(pub StarkHash);
 #[derive(Debug, Copy, Clone, PartialEq, Serialize, Deserialize)]
 pub struct ContractStateHash(pub StarkHash);
 
+/// A StarkNet contract's class hash.
+#[derive(Debug, Copy, Clone, PartialEq, Deserialize, Serialize)]
+pub struct ContractClassHash(pub StarkHash);
```

Comment:
Is this different to `ContractHash`? I would expect they're the same thing; although this naming is more accurate now.

Reply:
I've read the [article again](https://starkware.notion.site/Contracts-As-Classes-641060360aef4d048f7cb172afd57866) and yep, you are right, thanks for spotting :+1: